meson.build

```diff
@@ -935,9 +935,11 @@ if features['direct3d']
 endif
 
 drm = dependency('libdrm', version: '>= 2.4.105', required: get_option('drm'))
-features += {'drm': drm.found() and (features['vt.h'] or features['consio.h'] or features['wsdisplay-usl-io.h'])}
+libdisplay_info = dependency('libdisplay-info', required: get_option('drm'))
+features += {'drm': drm.found() and libdisplay_info.found() and
```

Comment:
So, it is mandatory dependency for drm now? Is it ok in practice?

Reply:
All distros are forced to carry this dependency for gnome and kde anyways. Should be fine.

Reply:
Would be useful to check what minimum version we can support

Reply:
It's a bit tricky here since the API technically isn't stable yet and may break in the future but I think for simplicity we can just set 0.2.0 for now. This PR doesn't use it but there are several `di_info_get` functions that were introduced and may potentially be useful.

Reply:
Turns out the pkgconfig version is 0.1.1 so I set it to that.